README.md

```diff
@@ -76,6 +76,8 @@ my_credential:
     google_json_key_location: /path/to/my-project-xxxxxxxxxxxx.json
 ```
 
+Or simply `cat .fog.example >> ~.fog` and edit parameters in the file.
```

Comment:
`~.fog` is an incorrect glob, as it'll resolve to: /Users/myuser.fog, you need a slash: `~/.fog`